### ISSUE TITLE
indentation rule should allow for intended multi-line values under the property

### ISSUE THREAD

**Issue by FiniteLooper** [sasstools/sass-lint#1190]
I am running Sass Lint version `1.12.1` from the command line, and from the VSCode IDE extension.  See the following code, where I get errors from the `indentation` rule saying 
```
Expected indentation of 4 spaces but found 6.
```

### Source code

```scss
  .card-body {
    min-height: 185px;
    animation: shine 1s infinite;

    background-image:
      linear-gradient(100deg, $skeleton-highlight-bg1, $skeleton-highlight-bg2 50%,$skeleton-highlight-bg1 80%),
      linear-gradient($skeleton-content-color $skeleton-content-height, transparent 0),
      linear-gradient($skeleton-content-color $skeleton-content-height, transparent 0),
      linear-gradient($skeleton-content-color $skeleton-content-height, transparent 0),
      linear-gradient($skeleton-content-color $skeleton-content-height, transparent 0);

    background-repeat: repeat-y;

    background-size:
      50px 200px, //highlight
      50% 200px,
      90% 200px,
      80% 200px,
      75% 200px;

    background-position:
      0 0, //highlight
      $skeleton-content-left ($skeleton-content-spacing * 1),
      $skeleton-content-left ($skeleton-content-spacing * 2),
      $skeleton-content-left ($skeleton-content-spacing * 3),
      $skeleton-content-left ($skeleton-content-spacing * 4);
  }
```

### Reasoning
It seems strange to expect very complex comma-separated properties like this with the values at the same indentation level as the property name:
```scss
background-size:
50px 200px, //highlight
50% 200px,
90% 200px,
80% 200px,
75% 200px;
```

or like this all on one line _(plus I lose my comments like this)_:
```scss
background-size: 50px 200px, 50% 200px, 90% 200px, 80% 200px, 75% 200px;
```

They way I have it written in the first snippet is far more readable and it should be allowed for property values that take up more than one line.  Writing them in the other two ways that this rule wants it a bit too strict.

See the screenshot below for what is show in the IDE.  Each green underline is a separate violation.

![image](https://user-images.githubusercontent.com/463685/37829287-d86ca89e-2e74-11e8-9edd-0a6cb0041fa7.png)

**Comment by scrimothy** [comment]
Completely agree! Also, just we we're not painting ourselves into a "comma separated values" corner, I expect space separated values to be handled the same way (e.g. border-radius, margin, padding, etc).

```
border-radius:
      ($border-radius - 2px)
      ($_large-radius - $small-padding - 2px)
      ($_large-radius - $small-padding - 2px);
```

**Comment by ChrisMBarr** [comment]
I assume this has been dealt with by this point - it's been several years now so I'm closing this issue to get it out of my list just like I closed the related issue today as well
